core/sta/ecc_self_tests.c

```diff
@@ -208,6 +373,8 @@ static TEE_Result invoke_command(void *pSessionContext __unused,
 	switch (nCommandID) {
 	case CMD_ECC_GEN_KEY_SELF_TESTS:
 		return ecc_generate_key_tests(pParams[0].value.a);
+	case CMD_ECC_DSA_TESTS:
+		return ecc_dsa_tests(pParams[0].value.a);
```

Comment:
This was one comment from Jens on the ECC Gen key. As the patch is at crypto_ops level, we cannot add test the feature at GP level yet (as in xtest 400x). This will be done in further patches. We may then remove this self static TA used for testing crypto_ops.

The client part of the test exist on the linaro-private git, branch ecc_next. This private git will become obsolete soon. So I'll create a pull-request on https://github.com/OP-TEE/optee_test soon.